controller/transfer/project_transfer_manager.py

```diff
@@ -611,14 +612,33 @@ def import_file(
                 ),
             )
 
+    def __transform_embedding_by_name(embedding_name: str):
+        splitted_name = embedding_name.split("-")
+        attribute_name = splitted_name[0]
+        embedding_type = splitted_name[1]
+        model = "-".join(splitted_name[2:])
+        if "bag-of-words" == model or "bag-of-characters" == model or "tf-idf" == model: 
+            platform= enums.EmbeddingPlatform.PYTHON.value
+        else:
+            platform = enums.EmbeddingPlatform.HUGGINGFACE.value
+        name = f"{attribute_name}-{embedding_type}-{platform}-{model}"
+        return platform, model, name
```

Comment:
why is this a function inside a function?

Reply:
It is only used at this place